### ISSUE TITLE
Support for teams

### ISSUE THREAD

**Issue by BoscoDomingo** [httpie/httpie#1514]
## Checklist

- [X] I've searched for similar feature requests.

---

## Enhancement request

Simple: enable syncing of workspaces between teams, so the same Request can be edited by one user and used by another. This can be easily set up in a GitHub repo so it has no cost to you.

---

## Problem it solves

Ability to work as a team without having to manually share requests

---

## Additional information, screenshots, or code examples

Postman and Insomnia both offer this, and it's quite a valuable feature. It's keeping me from using HTTPie. 

It could also open the door to shared environments.

**Comment by ScottRFrost** [comment]
We've worked around this internally by creating a Github shared / service account that we all log on to in HTTPie desktop, but having actual shared / sync would be amazing.

Alternatively, provide a simple way to save / load groups and requests from filesystem, then we could just sync those files via Git / OneDrive / etc.

**Comment by kalashnikovisme** [comment]
Saving requests and collections to the filesystem will be enough for the beginning. Wanna suggest using HTTPie for every client, but it's declined every time because of any sharing absent :slightly_frowning_face:

**Comment by BoscoDomingo** [comment]
Yeah, so long as we can get Git going, that would be great for many small teams and to encourage wider adoption. Right now we've opted for Bruno due to this

**Comment by nimr77** [comment]
any update on this ?

**Comment by anujkumar-df** [comment]
No update?

**Comment by ScottRFrost** [comment]
@anujkumar-df @nimr77 we actually gave up on waiting for httpie GUI to implement this and switched to https://github.com/usebruno/bruno
